rdr/busybox/app-busybox-1/subscriptions/busybox/drpc.yaml

```diff
@@ -6,9 +6,11 @@ metadata:
   labels:
     app: busybox-sample
 spec:
-  preferredCluster: "PLACEHOLDER-C1-ClusterName"
+  #preferredCluster: "PLACEHOLDER-C1-ClusterName"
+  preferredCluster: "sagrawal-c1"
   drPolicyRef:
-    name: PLACEHOLDER-DRPOLICY-NAME
+    #name: PLACEHOLDER-DRPOLICY-NAME
+    name: odr-policy-5m 
```

Comment:
Needs to be reverted.

Reply:
leftover from testing. I will clean it in final push.